docs/faq.md

```diff
@@ -290,9 +252,11 @@ If your `~/.specter` folder is gone and only one of your devices is lost
 
 When using Specter and importing an old wallet you would need to re-scan the blockchain in the wallet settings page.
 
+The labeld which you set within Specter will only reappear if you restore a Specter-backup, though.
+
 ### *To recover a multisig that was built on specter (eg: 2 of 3 with ColdCard), is having the seeds of all 3 signing wallets sufficient or do we need to backup more info?*
 
-Having seeds is enough, but in case you lose one of the seeds it is also **highly recommended** that you also backup your XPUBs. You can go to the wallet settings and export it as json file, this file has all the information needed to find your funds. "Export to wallet" software should give you one json file with all information needed for the recovery of your watch only wallet later on. 
+Having seeds is enough, but in case you lose one of the seeds it is also **mandatory** that you also backup your XPUBs. You can go to the wallet settings and export it as json file, this file has all the information needed to find your funds. "Export to wallet" software should give you one json file with all information needed for the recovery of your watch only wallet later on. 
```

Comment:
```suggestion
Having seeds is enough, but in case you lose one of the seeds it is also **mandatory** that you also backup your XPUBs. You can go to the wallet settings and export it as json file, this file has all the **public** key information needed. "Export to wallet" software should give you one json file with all information needed for the recovery of your watch only wallet later on. 
```